docs/api/quickstart.md

```diff
@@ -0,0 +1,138 @@
+
+## API Kubectl Example
+
+1. Set your KUBECONFIG environmental variable to the kubeconfig file you just created:
+
+  ```bash
+  export KUBECONFIG=$(pwd)/kubeconfig.yaml
+  ```
+
+  :::tip
+
+  Use `kubectl explain` to view the available fields for projects, or complex sub-fields of resources:
+
+  ```bash
+  kubectl explain projects
+  kubectl explain projects.spec
+  ```
+
+  :::
+
+2. Add the following content to a file named `project.yaml`:
+
+  ```yaml
+  apiVersion: management.cattle.io/v3
+  kind: Project
+  metadata:
+    # name should be unique accross all projects in every cluster
+    name: p-abc123
+    # generateName can be used instead of `name` to randomly generate a name.
+    # generateName: p-
+    # namespace should match spec.ClusterName.
+    namespace: local
+  spec:
+    # clusterName should match `metadata.Name` of the target cluster.
+    clusterName: local
+    description: Example Project 
+    # displayName is the human-readable name and is visible from the UI.
+    displayName: Example
+  ```
+
+3. Create the project:
+
+  ```bash
+  kubectl create -f project.yaml
+  ```
+
+4. Delete the project:
+
+  How you delete the project depends on how you created the project name.
+
+  **A. If you used `name` when creating the project**:
+
+  ```bash
+  kubectl delete -f project.yaml
+  ```
+
+  **B. If you used `generateName`**:
+
+    Replace `$PROJECT_NAME` with the randomly generated name of the project displayed by Kubectl after you created the project.
+
+    ```bash
+    kubectl delete project $PROJECT_NAME -n local
+    ```
+
+## Specifying CA Certs
+
+To ensure that your tools can recognize Rancher's CA certificates, most setups require additional modifications to the above template.
+
+1. In the upper left corner, click **☰ > Global Settings**. 
+2. Find and copy the value in the `ca-certs` field.
+3. Save the value in a file named `rancher.crt`.
+4. The following commands will convert `rancher.crt` to base64 output,  trim all new-lines, and update the cluster in the kubeconfig with the certificate, then finishing by removing the `rancher.crt` file:
+
+  ```bash
+  export KUBECONFIG=$PATH_TO_RANCHER_KUBECONFIG
+  kubectl config set clusters.rancher.certificate-authority-data $(cat rancher.crt | base64 -i - | tr -d '\n')
+  rm rancher.crt
+  ```
+5. (Optional) If you use self-signed certificatess that aren't trusted by your system, you can set the insecure option in your kubeconfig with kubectl:
+
+  :::warning 
+
+  This option shouldn't be used in production as it is a security risk.
+
+  :::
+
+  ```bash
+  export KUBECONFIG=$PATH_TO_RANCHER_KUBECONFIG
+  kubectl config set clusters.rancher.insecure-skip-tls-verify true
+  ```
+
+  If your Rancher instance is proxied by another service, you must extract the certificate that the service is using, and add it to the kubeconfig file, as demonstrated above.
```

Comment:
I don't think this is should be a subtext of step 5, but more of a `6. (Optional) If your Rancher...`. @MbolotSuse can you confirm?

Reply:
I'm not sure if I agree. If there were a new set of instructions I might agree, but this is really just saying "If you are using a proxy, you need to get the cert from the proxy and not the settings page". I think the flow would also look a bit awkward, in the proxy case you would just start at step 3 after getting the cert.

Reply:
Gotcha, but it should be moved out of step 5 correct?

Reply:
I think marty's suggestion is the best that I can think of. Really this is just a note that this section doesn't apply in proxy scenarios.